Framework/PythonInterface/mantid/api/src/Exports/IPeak.cpp

```diff
@@ -87,9 +87,16 @@ void export_IPeak() {
       .def("getL", &IPeak::getL, arg("self"), "Get the L index of the peak")
       .def("getHKL", &IPeak::getHKL, arg("self"),
            "Get HKL as a :class:`~mantid.kernel.V3D` object")
+      .def("getSamplePos", &IPeak::getSamplePos, arg("self"),
+           "Get the cached samplePos as a :class:`~mantid.kernel.V3D` object")
       .def("setHKL", (void (IPeak::*)(double, double, double)) & IPeak::setHKL,
            (arg("self"), arg("h"), arg("k"), arg("l")),
            "Set the HKL values of this peak")
+      .def("setSamplePos",
```

Comment:
Is there a reason why setting the sample position with V3D or numpy array is not exposed? Or is there an issue about that?